Jenkinsfile

```diff
@@ -455,7 +417,7 @@ pipeline {
         axes {
           axis {
             name 'DB'
-            values 'postgresql_96', 'mariadb_103'
+            values 'postgresql_96', 'postgresql_94', 'postgresql_107'
           }
           axis {
             name 'PROFILE'
```

Comment:
Let's rename the 'webapps-unit', 'webapps-unit-authorizations' to 'webapp-unit', 'webapp-unit-authorizations' (removing the `s`)